nms/app/packages/magmalte/app/views/equipment/EquipmentGateway.js

```diff
@@ -291,9 +291,13 @@ function GatewayTableRaw(props: WithAlert) {
                 </Link>
               ),
             },
-            {title: 'enodeBs', field: 'num_enodeb', type: 'numeric'},
-            {title: 'Subscribers', field: 'num_subscribers', type: 'numeric'},
-            {title: 'Health', field: 'health'},
+            {
```

Comment:
are we not modifying the width in the "upgrade" action table above ?

Reply:
Yep, fixed!